### ISSUE TITLE
Set-SPOOrgAssetsLibrary parameter OrgAssetType

### ISSUE THREAD

**Issue by Markus-Hanisch** [MicrosoftDocs/office-docs-powershell#6180]
Hello,

I wanted to set a document library as OrgAssetsLibrary of OrgAssetType "OfficeTemplateLibrary" as described in **[Example 3](https://docs.microsoft.com/en-us/powershell/module/sharepoint-online/set-spoorgassetslibrary?view=sharepoint-ps#examples)**:
>**Example 3**
>This example changes the OrgAssetType to OfficeTemplateLibrary.
>```powershell
>Set-SPOOrgAssetsLibrary -LibraryURL sites/branding/Templates -OrgAssetType OfficeTemplateLibrary
>```

However, the PS Cmdlet does not offer to set this parameter.
I have updated to the latest available version of the PowerShell module:

```powershell 
Get-InstalledModule -Name Microsoft.Online.SharePoint.PowerShell | Select Name, Version

Name                                   Version         
----                                   -------         
Microsoft.Online.SharePoint.PowerShell 16.0.20122.12000
```

Please support. How can I set the OrgAssetType to "OfficeTemplateLibrary" ?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f000145c-fc07-8254-a1f2-89173ff62906
* Version Independent ID: 2c515e74-cb12-7a5a-f059-7161049223b9
* Content: [Set-SPOOrgAssetsLibrary (sharepoint-online)](https://docs.microsoft.com/en-us/powershell/module/sharepoint-online/set-spoorgassetslibrary?view=sharepoint-ps)
* Content Source: [sharepoint/sharepoint-ps/sharepoint-online/Set-SPOOrgAssetsLibrary.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/sharepoint/sharepoint-ps/sharepoint-online/Set-SPOOrgAssetsLibrary.md)
* Service: **sharepoint-powershell**
* GitHub Login: @Maralesfahanpoor
* Microsoft Alias: **maesfaha**

**Comment by Maralesfahanpoor** [comment]
Hello Markus,

I am not exactly sure that what is the issue here, 
Is it that the PowerShell doesn't recognize the Set-SPOOrgAssetsLibrary cmdlet or it doesn't recognize the OrgAssetType parameter?
The Set-SPOOrgAssetsLibrary is for changing the existing document libraries properties like thumbnail and OrgAssetstype, Which means you should already have the OrgAssets document libraries and with this cmdlet you can change or set the Thumbnail and OrgAssetsType.

If you don't have any document libraries set up please follow the instruction in the [Add-SPOOrgAssetsLibrary](https://docs.microsoft.com/en-us/powershell/module/sharepoint-online/add-spoorgassetslibrary?view=sharepoint-ps) and follow example 2.

**Comment by Markus-Hanisch** [comment]
@Maralesfahanpoor 
The cmdlet does not recognize the parameter OrgAssetType.
When I try to add a document library as OfficeTemplateLibrary, I receive an error. 
The following PS cmdlet:
```powershell
$tenant = "yourtenantname"
Connect-SPOService -Url https://$tenant-admin.sharepoint.com
Add-SPOOrgAssetsLibrary -LibraryURL https://$tenant.sharepoint.com/sites/branding/OfficeTemplates -OrgAssetType OfficeTemplateLibrary
```
..results in:
```
Add-SPOOrgAssetsLibrary : A parameter cannot be found that matches parameter name 'OrgAssetType'.
At line:1 char:97
+ ... .sharepoint.com/sites/branding/OfficeTemplates -OrgAssetType OfficeT ...
+                                                     ~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Add-SPOOrgAssetsLibrary], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.Online.SharePoint.PowerShell.AddSPOOrgAssetsLibrary
```

**Comment by get-itips** [comment]
@Markus-Hanisch 
I am using this version

`Version : 16.0.20212.12000`

Which is higher than the one you report and I can use the OrgAssetsType param

```
 ⨯ agorz@PENDULUM  ~  Add-SPOOrgAssetsLibrary -LibraryURL https://m365xxxx.sharepoint.com/sites/Brand/OfficeTemplates -OrgAssetType OfficeTemplateLibrary

Confirm
Are you sure you want to perform this action?
Performing the operation "Add a new Tenant CDN origin" on target
"https://m365x936743.sharepoint.com/sites/Brand/OfficeTemplates".
[Y] Yes  [A] Yes to All  [N] No  [L] No to All  [S] Suspend  [?] Help (default is "Y"): Y

Document library added successfully.
```

Hope this info helps you.

**Comment by Markus-Hanisch** [comment]
@get-itips 
Okay, sounds promising :)
I will update to the latest version and try again. :)

**Comment by yogkumgit** [comment]
@get-itips  Thank you very much for the contribution and sharing this explanation. @Markus-Hanisch Hope this comment is helpful for you. If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Comment by Markus-Hanisch** [comment]
@get-itips 
Hmm strange... I have updated to the latest version.
_Side note: 
I have installed several versions of this module on that machine. (dependencies of the Microsoft365DSC module)_
I imported the latest version by `Import-Module` cmdlet. The parameter `OrgAssetsType` does not show up.

So I used another machine, updated to the latest version of the module and the parameter showed up as expected.

Lessons learned: 
no parallel module versions - I expected when I Import a specific version, I can use it as if I would have installed the latest version only (?)
